XamlControlsGallery/SettingsPage.xaml.cs

```diff
@@ -129,5 +140,19 @@ private void navigationLocation_SelectionChanged(object sender, SelectionChanged
         {
             NavigationOrientationHelper.IsLeftMode = navigationLocation.SelectedIndex == 0;
         }
+
+        private async void FolderButton_Click(object sender, RoutedEventArgs e)
+        {
+            FolderPicker folderPicker = new FolderPicker();
+            folderPicker.SuggestedStartLocation = PickerLocationId.PicturesLibrary;
+            folderPicker.FileTypeFilter.Add(".png");

```

Comment:
We need this because otherwise the file picker crashes right? Maybe we could add a comment for that?

Reply:
Yes, it's entirely meaningless as far as I can tell, but if you don't add at least one file type it gets sad. I added a comment.